### PR TITLE
fix string errors

### PR DESCRIPTION
**Pull request by djredhand**
Put some tests on there. Things should look much better now.

### CODE REVIEW

**Comment by thraxil** [comment]
I'm not going to merge any more PRs in this project that don't come with tests.

File: uelc/main/tests/test_models.py [hunk 1]
```diff
@@ -1,6 +1,63 @@
 from django.test import TestCase
+#from django.core.exceptions import ValidationError
```
Comment (sdreher) [inline uelc/main/tests/test_models.py]: Good idea to remove commented code.

File: uelc/main/views.py [hunk 2]
```diff
@@ -274,6 +274,8 @@ def get(self, request, path):
             library_items=self.get_library_items(case),
         )
         context.update(self.get_extra_context())
+        import pdb
```
Comment (sdreher) [inline uelc/main/views.py]: pdb statements must be removed before a merge.

Comment (djredhand) [inline uelc/main/views.py]: oops, yea that was not supposed to be committed.

File: uelc/main/tests/test_models.py (hunk 1 above)
Comment (djredhand) [inline uelc/main/tests/test_models.py]: I like having that in there because I am pretty sure I am going to need it soon. But I will delete it for now.

Comment (sdreher) [inline uelc/main/tests/test_models.py]: In general, commented out code is considered bad practice. Here's the whys/wherefores: http://www.markhneedham.com/blog/2009/01/17/the-danger-of-commenting-out-code/.

**Pull request closed without merging**